documentation/source/PoseEstimation.md

```diff
@@ -0,0 +1,187 @@
+However, our implementation does NOT include computation of AP/AR scores per area range. It also natively support evaluation in DDP mode. 
+
+It is worth noting that usually reported AP/AR scores in papers are obtained using TTA (test-time augmentation) and additional postprocessing on top of the main model. 
+
+A horizontal flip is a common TTA technique that is used to increase accuracy of the predictions at the cost of running forward pass twice. 
+Second common technique is a multi-scale approach when one perform inference additionally on 0.5x and 1.5x input resolution and aggregate predictions. 
+
+When training model using SuperGradients, we use neither of these techniques. If you want to measure AP/AR scores using TTA you may want to write your own evaluation loop for that.
+
+In order to use `PoseEstimationMetrics` you have to pass a so-called `post_prediction_callback` to the metric, which is responsible for postprocessing of the model's raw output into final predictions. 
+
+### Postprocessing
+
+Postprocessing refers to a process of transforming the model's raw output into final predictions. Postprocessing is also model-specific and depends on the model's output format. 
+For `DEKR` model, the postprocessing step is implemented in [DEKRPoseEstimationDecodeCallback]((https://docs.deci.ai/super-gradients/docstring/training/utils/#training.utils.pose_estimation.dekr_decode_callbacks.DEKRPoseEstimationDecodeCallback)) class. 
+When instantiating the metric, one has to pass a postprocessing callback as an argument:
+
+```yaml
+training_hyperparams:
+    valid_metrics_list:
+      - PoseEstimationMetrics:
+          num_joints: ${dataset_params.num_joints}
+          oks_sigmas: ${dataset_params.oks_sigmas}
+          max_objects_per_image: 20
+          post_prediction_callback:
+            _target_: super_gradients.training.utils.pose_estimation.DEKRPoseEstimationDecodeCallback
+            max_num_people: 20
+            keypoint_threshold: 0.05
+            nms_threshold: 0.05
+            nms_num_threshold: 8
+            output_stride: 4
+            apply_sigmoid: False
+```
+
+### Visualization
+
+Visualization of the model predictions is a very important part of the training process for pose estimation models. 
+By visualizing the predicted poses, developers and researchers can identify errors or inaccuracies in the model's output and adjust the model's architecture or training data accordingly.
+
+Overall, visualization is an important tool for improving the accuracy and usability of pose estimation models, both during development and in real-world applications.
+
+SuperGradients provide an implementation of `DEKRVisualizationCallback` to visualize predictions for `DEKR` model. 
+You can use this callback in your training pipeline to visualize predictions during training. To enable this callback, add the following lines to your training YAML recipe:
```

Comment:
Maybe a word on where will it appear ?
It would also be a good idea to point to SG loggers docs I believe.

Reply:
Also, if not too much work, it would be nice to add a photo of the output visualisation.

Reply:
Agree. Added